### ISSUE TITLE
Are passes generated from UK Vaccination Certificates accepted in the EU ?

### ISSUE THREAD

**Issue by ieb** [clawfire/covid19-passbook-generator#305]
### Is your feature request related to a problem? Please describe.

This is a question more than a feature request.

The app works perfectly when I use my QR code from my UK NHS Issued certificate. All the information is extracted correctly with no errors.  However, do you know if the generated pass will be accepted in the EU or is there still a dispute between the EU and UK over accepting NHS Digital issued vaccination status ?

### Describe the solution you'd like

That I can use this app to prove my vaccination status in the EU when travelling rather than having to use a paper copy or the NHS App.

### Anything else?

btw I did not vote leave, and still think it was a stupid idea ;) 

A friend in Luxembourg who has been UK vaccinated (student in the UK) sent me the link to the site after using it and travelling so I am hoping the answer is yes.

**Comment by clawfire** [comment]
Hello there 😃  Thanks for trying this tool, and thanks to your friends to share it 😄 

TL;DR; Yes it will work. 

Long answers: 

Ok so, we don't transform anything. We are just reading the QRcode, extracting the information to display them on a card with the QRcode that we just "copy-paste" on it. There's no transformation at all from your data. It's not a new code or a new format.

And yes, we are only supporting the "EU" standard right now (I still hope find time to work on US, NZ, AU ones). NHS follows the same standard (which is a good idea for letting British people travel across Europe) from day one. But since they are not part of the EU anymore, the digital keys used to sign the certificate were not in the EU system. Just like for Switzerland. It was up to individual countries to decide if they wanted to support that extra set of keys. 

Luckily, since Oct. 29th, the EU opened its system to non-EU countries. The UK could insert its keys and make every NHS pass recognizable in EU countries and EU Digital Covid Certificate partner countries.

**Comment by clawfire** [comment]
Oh! BTW I have only the "EU skin" right now and didn't find any solution to make like ... NHS version or other versions without having to rewrite a lot of specific code, so ... Hope you like the EU-themed card 😄
